aiogram/api/types/user.py

```diff
@@ -33,6 +41,80 @@ class User(TelegramObject):
 
     @property
     def full_name(self) -> str:
+        """
+        Get full name of user.
+        :return: str
+        """
         if self.last_name:
             return f"{self.first_name} {self.last_name}"
         return self.first_name
+
+    @property
+    def mention(self) -> str:
+        """
+        Get user's username to mention him.
+        Full name will be returned if user has no username.
+        :return: str
+        """
+        if self.username:
+            return "@" + self.username
+        return self.full_name
+
+    @property
+    def locale(self) -> Optional[babel.core.Locale]:
+        """
+        Get user's locale
+        :return: :class:`babel.core.Locale`
+        """
+        if not self.language_code:
+            return None
+        if not hasattr(self, "_locale"):
+            setattr(self, "_locale", babel.core.Locale.parse(self.language_code, sep="-"))
+        return getattr(self, "_locale")
+
+    @property
+    def url(self) -> str:
+        """
+        Get user's profile url.
+        :return: str
+        """
+        return f"tg://user?id={self.id}"
+
+    def get_mention(self, name: Optional[str] = None, as_html: Optional[bool] = None) -> str:
+        """
+        Get user's mention url.
+
+        :param name: Name of user in the mention link. User's full_name property will be used if not specified.
+                     Defaults to None
+        :param as_html: Boolean flag defining format of the resulting mention. Bot parse_mode property will be used if
+                        not specified. Defaults to None
+
+        :return: str
+        """
+        if (
+            as_html is None
+            and self.bot.parse_mode
+            and self.bot.parse_mode.upper() == ParseMode.HTML
+        ):
+            as_html = True
+
+        if name is None:
+            name = self.full_name
+        if as_html:
+            return markdown.hlink(name, self.url)
+        return markdown.link(name, self.url)
+
+    async def get_profile_photos(
```

Comment:
Aliases for API methods should NOT be awaitable in due to new design of API methods.
See examples for aliases in Message object.

PS: https://t.me/aiogram_ru/219716